default/templates/default/base-tool.html

```diff
@@ -38,6 +38,15 @@
     <span class="file open-drive-link hidden">
         {% trans "File saved on Google Drive" %}&nbsp;<a target="_blank" role="button"> {% trans "Open" %}</a>
     </span>
+    <div class="form-group form-inline form-send hidden">
+        <label>{% trans "Send result to function:" %}</label>
+        <select class="form-control to-function">
+            <option id="compile" value="/compile/">{% trans "Compile Releases" %}</option>
+            <option id="upgrade" value="/upgrade/">{% trans "Upgrade from 1.0 to 1.1" %}</option>
+            <option id="convert" value="/to-spreadsheet/">{% trans "Convert to CSV/Excel" %}</option>
+        </select>
+        <a class="send-button" href="#" role="button">{% trans "Send" %}</a>
+    </div>
```

Comment:
Creo que, en lugar de tener estas opciones de manera estática aquí (y usar javascript para esconder opciones!) sería mejor que cada feature tenga una lista de opciones permitidas que se le pase al template. Entonces se podría usar un `for` aquí en el template para renderizar cada opción.

Otra opción que me parece mejor sería definir un [inclusion tag](https://docs.djangoproject.com/en/2.2/howto/custom-template-tags/#inclusion-tags) que reciba como parámetro las opciones a desplegar. Así no va a ser necesario tener la lista de opciones permitidas en views, sino que en el template mismo se le van a poder pasar las funciones.